### ISSUE TITLE
a Sanic extension that generates APIs from python function and classes.

### ISSUE THREAD

**Issue by yunstanford** [channelcat/sanic#775]
I have a transmute framework for sanic. basically, it is a sanic extension that generates APIs from python function and classes. and it's very easily integrate with Swagger with zero effort, just do 

```
# add swagger
add_swagger(app, "/api/v1/swagger.json", "/api/v1/")
```

find out more here:
http://sanic-transmute.readthedocs.io/en/latest/

**Comment by seemethere** [comment]
Would you like to submit a PR to add this to the extensions documentation? https://github.com/channelcat/sanic/blob/master/docs/sanic/extensions.md

**Comment by seemethere** [comment]
Closed with #782
